src/pudl/extract/extractor.py

```diff
@@ -145,7 +156,7 @@ def process_raw(
         self, df: pd.DataFrame, page: str, **partition: PartitionSelection
     ) -> pd.DataFrame:
         """Takes any special steps for processing raw data and renaming columns."""
-        return df
+        return df.rename(columns=self._metadata.get_column_map(page, **partition))
```

Comment:
previously only the excel extractor actually did the rename by default. it seems like most of the new csv extracted raw inputs were not trying to rename any of the columns, but still have column_maps w/ 1:1 existing name : pudl name. So as far as i can tell nothing has changed.

Reply:
Tested to make sure that if we delete the column mapping CSV for datasets where it actually isn't necessary everything works just fine in the extraction. So this seems like a good and fine decision.